modules/src/ics02_client/client_def.rs

```diff
@@ -14,8 +14,8 @@ use crate::ics23_commitment::commitment::{CommitmentPrefix, CommitmentProof, Com
 use crate::ics24_host::identifier::{ClientId, ConnectionId};
 
 use ::tendermint::block::Height;
-
 use prost_types::Any;
+
 use std::convert::TryFrom;
 use tendermint_proto::{DomainType, Error, Kind};
 #[cfg(test)]
```

Comment:
Small nitpick: Can we group the imports as follows?

```rust
use prost_types::Any;
use std::convert::TryFrom;

use ::tendermint::block::Height;
use tendermint_proto::{DomainType, Error, Kind};

 #[cfg(test)]
```

and perhaps move the `std` and `prost` imports at the top of the file together with the `serde_derive` import.